pkg/controller/pod.go

```diff
@@ -730,48 +815,89 @@ func (c *Controller) getPodAttachmentSubnet(pod *v1.Pod) ([]*kubeovnv1.Subnet, e
 		return nil, err
 	}
 
-	result := make([]*kubeovnv1.Subnet, 0, len(attachments))
+	result := make([]*kubeovnNet, 0, len(attachments))
 	for _, attach := range attachments {
-		provider := fmt.Sprintf("%s.%s", attach.Name, attach.Namespace)
+		networkClient := c.config.AttachNetClient.K8sCniCncfIoV1().NetworkAttachmentDefinitions(attach.Namespace)
```

Comment:
We prefer to use informer lister cache than client directly calls to enhance performance. However if use informer we require that the attachment crd exists otherwise it will fail to start the informer, users who haven't install multus will panic.

Reply:
We can install multus crds by default later

Reply:
the best case scenario is to install multus by default, then we can use informerlister cache herer, but I'm afraid it won't be very friendly to users who don't need multus.